src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -380,6 +383,44 @@ public void SarifErrorListItem_ResultMessageFormat_NoTrailingPeriod()
             item.ShortMessage.Should().Be(s1);
         }
 
+        [Fact]
+        public void SarifErrorListItem_ResultMessageFormat_LongTextWithoutBreak()
+        {
+            // very long text without break like space or NewLine, longer than default max lengh 165
+            const string s1 = "1234567890";
+            var result = new Result
+            {
+                Message = new Message()
+                {
+                    Text = string.Concat(Enumerable.Repeat(s1, 20)), // replace s1 10 times, a string which length is 200
+                },
+            };
+
+            SarifErrorListItem item = MakeErrorListItem(result);
+            int breakPoistion = 165;
+            item.ShortMessage.Length.Should().Be(breakPoistion + 2);
+            item.Message.Length.Should().Be(200 - breakPoistion);
+        }
+
+        [Fact]
+        public void SarifErrorListItem_ResultMessageFormat_LongTextWitBreak()
```

Comment:
>SarifErrorListItem_ResultMessageFormat_LongTextWitBreak [](start = 20, length = 55)

can you create a test using environment new line, \r and \n? that would check the method we updated